docs/configuration/index.md

```diff
@@ -787,6 +787,14 @@ All Druid components can communicate with each other over HTTP.
 |`druid.global.http.unusedConnectionTimeout`|The timeout for idle connections in connection pool. The connection in the pool will be closed after this timeout and a new one will be established. This timeout should be less than `druid.global.http.readTimeout`. Set this timeout = ~90% of `druid.global.http.readTimeout`|`PT4M`|
 |`druid.global.http.numMaxThreads`|Maximum number of I/O worker threads|`max(10, ((number of cores * 17) / 16 + 2) + 30)`|
 
+### Common endpoints Configuration
+
+This section contains the configuration options for endpoints that are supported by all processes.
+
+|Property| Description                                                                                                                                  |Default|
+|--------|----------------------------------------------------------------------------------------------------------------------------------------------|-------|
+|`druid.server.hiddenProperties`| If property names or substring of property names (case insensitive) is in this list, responses of the `/status/properties` endpoint do not show these properties |`["druid.s3.accessKey","druid.s3.secretKey","druid.metadata.storage.connector.password"]`|
```

Comment:
I want to raise the idea of adding new default generic property substrings here such as `password` and `key` to help ensure redaction of any future sensitive properties. I just checked a local deploy and there is definitely other sensitive properties that are being exposed due to the restricted default set + hidden config (until now)

Reply:
Sure thing, we can already include substrings already at `druid.server.hiddenProperties `. I'd propose to add those substrings mentioned at https://github.com/apache/druid/issues/12063: `["password", "key", "token", "pwd"]`

Reply:
In this case I'll also add these substrings to `common.runtime.properties` which are already in the codebase.